EventFilter/Utilities/plugins/BuildFile.xml

```diff
@@ -8,8 +8,10 @@
   <use   name="FWCore/Sources"/>
   <use   name="FWCore/Utilities"/>
   <use   name="IOPool/Streamer"/>
+  <use   name="DQMServices/Core"/>
```

Comment:
@smorovic the addition of the DQM-related plugin adds this dependency that misses in the whole rest of the EventFilter area. Would not be possible to locate that plugin into a DQM package? At present the most obvious location I may see without creating a new package just for one class is ```DQM/HLTEvF```.

Are  you expecting to have more DQM-oriented code in this package, or within EventFilter?

Reply:
@jfernan2 FYI

Reply:
@fabiocos, I agree, it does not have to go to EventFilter/Utilities. DQM/HLTEvF to me seems as a good location, and there is no dependence on the rest of the EventFilter code.

We don't foresee to add more EventFilter-specific DQM code. Intention for this module is to  generate a lot of objects (and bins) to produce high CPU load by fastHadd in performance tests on DAQ hosts.